examples/cra-kitchen-sink/src/stories/welcome.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import { Welcome } from '@storybook/react/demo';
 import { storiesOf } from '@storybook/react';
-import { linkTo } from '@storybook/addon-links';
 
-storiesOf('Welcome', module).add('to Storybook', () => <Welcome showApp={linkTo('Button')} />);
+storiesOf('Welcome', module).add('to Storybook', () => <Welcome showKind="Button" />);
```

Comment:
do we need to update these in the cli templates?

Reply:
This was already updated. Actually I just did some copy-pasting from commented code in `stories/index.js` to partially restore the `release/3.3` changes.

If you feel like it, you can sync the rest of the changes in another PR

Reply:
Why does this need to change to begin with?

Reply:
It has changed in #1829 not now. It’basically adding a usage of a new feature

Reply:
It has changed in #1829 not now. It’basically adding a usage of a new feature

Reply:
Ok, well I'm OK with this change.